build.sbt

```diff
@@ -5,8 +5,18 @@ version := "0.1"
 scalaVersion := "2.13.1"
 
 // https://www.scala-sbt.org/1.x/docs/Testing.html#Integration+Tests
+lazy val EndToEndTest = config("e2e") extend(Test)
+lazy val endToEndTestSettings: Seq[Def.Setting[_]] =
+  inConfig(EndToEndTest)(Defaults.testSettings) ++
+    Seq(
+      scalaSource in EndToEndTest := baseDirectory.value / "src/e2e/scala",
```

Comment:
> Adults should have no more than 30g of free sugars a day, (roughly equivalent to 7 sugar cubes).

– NHS.uk (2017). How does sugar in our diet affect our health? [online] Available at: https://www.nhs.uk/live-well/eat-well/how-does-sugar-in-our-diet-affect-our-health/

Reply:
![](https://st3.depositphotos.com/6125608/17863/i/1600/depositphotos_178638016-stock-photo-small-yellow-toy-truck-loaded.jpg)